tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/Tcl.xtext

```diff
@@ -135,7 +135,7 @@ AEComparison:
 	right=AssertionInnerExpression)?;
 
 AEVariableReference:
-	variable=[AssignmentVariable] ( '.' ( key=ID | key=STRING ) )?;
+	'@'? variable=[AssignmentVariable] ( '.' ( key=ID | key=STRING ) )?;
```

Comment:
For grammar changes we should have a parser test, a change in the formatter and a respective formatter test.